cborg/src/Codec/CBOR/Pretty.hs

```diff
@@ -6,6 +6,8 @@
 {-# LANGUAGE StandaloneDeriving  #-}
 {-# LANGUAGE UnboxedTuples       #-}
 
+{-# OPTIONS_GHC -fno-warn-incomplete-patterns #-}
```

Comment:
That's a bit worrying. Which warnings does it silence?

Reply:
In this [case](https://github.com/well-typed/cborg/pull/223/files#diff-6ad34f0927f7ad58df60ef665fd70b7eL163) expression. The compiler complained about the missing cases within each constructor (which weren't relevant). I'm actually thinking I should leave the catch all in the above because we will have an issue if an additional constructor is added to `Tokens` and is not accounted for in `unconsToken`

Reply:
I agree that a catch-all is preferable to disabling warnings for the entire module – the catch-all silences the warnings only for a particular case expression!

I'll leave the further review to a maintainer who is actually familiar with this bit of code! ;)